config/routes.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  resources :users, only: %i[new create]
+  resources :users, only: %i[new create edit]
```

Comment:
We also need to include the `:update` route here. We need this route in order to send a POST request to update the user. This request is sent when you click in to update the user.
![Screenshot from 2019-12-18 23-15-55](https://user-images.githubusercontent.com/30977845/71139232-5f1dac00-21ec-11ea-97eb-9ccec0535888.png)